tasks/6.txt

```diff
@@ -1,3 +1,26 @@
 Create a query to return all products with category and supplier company names:
 
-| ProductName | CategoryName | SupplierCompanyName |
\ No newline at end of file
+| ProductName | CategoryName | SupplierCompanyName |
+
+db.products.aggregate([
+    { $lookup: {
+        from: 'categories',
+        localField: 'CategoryID',
+        foreignField: 'CategoryID',
+        as: 'cat',
+    }},
+    { $unwind: '$cat'},
+    { $lookup: {
+        from: 'suppliers',
+        localField: 'SupplierID',
+        foreignField: 'SupplierID',
+        as: 'sup',
+    }},
+    { $unwind: '$sup'},
+    { $project: {
+        _id:false,
+        ProductName: 1,
+        CategoryName: '$cat.CategoryName',
+        SupplierCompanyName: '$sup.CompanyName',
+    }}
+])
```

Comment:
looks fine, in general. But on big amount of data the query will be slow. 
Can be better if do projection inside of lookup with pipeline
https://docs.mongodb.com/manual/reference/operator/aggregation/lookup/#join-conditions-and-uncorrelated-sub-queries